django/views/templates/csrf_403.html

```diff
@@ -64,7 +64,7 @@ <h2>Help</h2>
     <code>csrf_protect</code> on any views that use the <code>csrf_token</code>
     template tag, as well as those that accept the POST data.</li>
 
-    <li>The form has a valid CSRF token. After logging in in another browser
+    <li>The form has a valid CSRF token. After logging in to another browser
```

Comment:
As far as I'm aware, users can login _"to an app"_, but not _"to a browser tab"_ :thinking: Maybe _"on another browser tab"_.

Reply:
Ah yeah, if I read the context I think felix is right with "on another browser tab".

Reply:
I see nothing wrong with the current wording. "in another tab" has about 103 million Google search results.

Reply:
It's not so much the "in another tab" it's the "in in" which is usually avoided 👍

Reply:
I thought maybe that was the argument. I still think it reads just fine, but I suppose "in a tab" or "on a tab" is just a matter of preference. I would at least not say it's "incorrect preposition use" unless there's a grammatical rule I'm unaware of.

Reply:
Based on the discussion here I updated the phrase to:  "After logging  **on another** browser tab".
Let me know if it works (sorry I'm not a native speaker). @timgraham @shangxiao @felixxm

Reply:
It would be "logging in on". I'm not sure it's worth changing. I've always thought of doing things "in" tabs, not "on" them. Google seems to return an equal number of results for "logging in in another tab" and "logging in on another tab".

Reply:
I don't think using double `in` is good here and that is what I want to fix.

Here is some more discussion about that: https://english.stackexchange.com/questions/371126/using-logging-in-correctly